.github/workflows/acceptance-tests.yml

```diff
@@ -64,11 +64,11 @@ jobs:
           backup_schedule:
             - 'mongodbatlas/**backup_schedule**.go'
           project:
+            - 'internal/service/project/*.go'
+            - 'internal/service/projectipaccesslist/*.go'
             - 'mongodbatlas/data_source_mongodbatlas_project_invitation*.go'
-            - 'mongodbatlas/fw_data_source_mongodbatlas_project_ip_access_list*.go'
             - 'mongodbatlas/resource_mongodbatlas_access_list_api_key*.go'
             - 'mongodbatlas/resource_mongodbatlas_project_invitation*.go'
```

Comment:
you need to remove the "mongodbatlas" part from files

Reply:
or use like in other places like: 'mongodbatlas/**access_list_api_key**.go' , etc.

Reply:
good point, had forgotten about that change